### PR TITLE
Reword command line option section

### PR DESCRIPTION
**Pull request by xerus2000**
Use imperative for the description and adjust grammar.

### CODE REVIEW

File: source/chapters/appendix.rst [hunk 1]
```diff
@@ -327,28 +324,26 @@ Option                  Description
 --locale LOCALE         Use a custom locale for loading translations
                         (e.g ``fr``)
 ----------------------  -------------------------------------------------
--f, --fullScreen        Starts Mixxx in full-screen mode
+-f, --fullScreen        Start Mixxx in full-screen mode-
```
Comment (rryan) [inline source/chapters/appendix.rst:327]: Was this supposed to end in a period?

**Comment by rryan** [comment]
add to whitelist

File: source/chapters/appendix.rst [hunk 2]
```diff
@@ -305,19 +305,16 @@ Option                  Description
...
+                        receives as well as script functions it loads.
 ----------------------  -------------------------------------------------
---developer             Enables developer-mode. Includes extra log info,
-                        stats on performance, a Developer tools menu,
-                        extra developer skins, and provides useful
-                        information for skin writers in widget tooltips
-                        and logs .
+--developer             Enable developer-mode. Includes extra logs, stats
+                        on performance, a Developer tools menu and extra
+                        developer skins. Also provides useful information
```
Comment (ronso0) [inline source/chapters/appendix.rst:313]: there are no "developer only" skins.. ;)

Comment (Be-ing) [reply source/chapters/appendix.rst:313]: Speaking of which, can we delete https://github.com/mixxxdj/developer_skins already?

Comment (ronso0) [reply source/chapters/appendix.rst:313]: Enable developer-mode. Includes extra logs, stats on performance and the Developer tools menu.
Also, logs and tooltips provide extra info for skin developers.

Comment (ronso0) [reply source/chapters/appendix.rst:313]: > Speaking of which, can we delete https://github.com/mixxxdj/developer_skins already?

doesn't hurt anyone, does it? Let's keep it until someone (me?) did the next rev of the skin creation docu.
Even then, should we shrink-pack it and put it somewhere for documentation?

Comment (Be-ing) [reply source/chapters/appendix.rst:313]: Documenting code that is years out of date would be a waste of time that could only create confusion

Comment (Be-ing) [reply source/chapters/appendix.rst:313]: We don't keep commented code around, so why should we keep that repository?

Comment (Holzhaus) [reply source/chapters/appendix.rst:313]: We can archive the repo for now (that makes it read-only), and delete when we're sure it contains nothing worth keeping.

File: source/chapters/appendix.rst (hunk 1 above)
Comment (xerus2000) [reply source/chapters/appendix.rst:327]: yes, thanks :)

File: source/chapters/appendix.rst (hunk 2 above)
Comment (xerus2000) [reply source/chapters/appendix.rst]: Adjusted the docs :)

Comment (ronso0) [reply source/chapters/appendix.rst]: > We can archive the repo for now (that makes it read-only), and delete when we're sure it contains nothing worth keeping.

Jup. I'll take a look when 2.3 is out the door.

File: source/chapters/appendix.rst [hunk 3]
```diff
@@ -305,19 +305,16 @@ Option                  Description
                         for user settings files such as the library
                         database and preferences configuration file.
 ----------------------  -------------------------------------------------
---controllerDebug       Causes Mixxx to display/log all controller data
-                        it sends and receives, and script functions it
-                        loads.
+--controllerDebug       Display/log all controller data Mixxx sends and
```
Comment (Be-ing) [inline source/chapters/appendix.rst:308]:
```suggestion
--controllerDebug       Log all controller data Mixxx sends and
```

File: source/chapters/appendix.rst [hunk 4]
```diff
@@ -305,19 +305,16 @@ Option                  Description
...
+                        receives as well as script functions it loads.
```
Comment (Be-ing) [inline source/chapters/appendix.rst:309]:
```suggestion
                        receives as well as scripts it loads.
```

File: source/chapters/appendix.rst [hunk 5]
```diff
@@ -305,19 +305,16 @@ Option                  Description
...
 ----------------------  -------------------------------------------------
---developer             Enables developer-mode. Includes extra log info,
-                        stats on performance, a Developer tools menu,
-                        extra developer skins, and provides useful
-                        information for skin writers in widget tooltips
-                        and logs .
+--developer             Enable developer-mode. Includes extra logs, stats
```
Comment (Be-ing) [inline source/chapters/appendix.rst:311]:
```suggestion
--developer             Enable developer mode. Includes extra logs, stats
```

File: source/chapters/appendix.rst [hunk 6]
```diff
@@ -305,19 +305,16 @@ Option                  Description
...
+                        on performance and the Developer tools menu as
+                        well as tooltips and logs useful for skin
+                        developers.
 ----------------------  -------------------------------------------------
---safeMode              Disables risky things at start up. This should
-                        help troubleshoot crashes at startup or if Mixxx
-                        won't boot.
+--safeMode              Disable risky things at start up. This should
+                        help troubleshoot startup crashes.
```
Comment (Be-ing) [inline source/chapters/appendix.rst:317]: Disable OpenGL widgets (scrolling waveforms, spinnies) to work around GPU driver bugs. If Mixxx is crashing on startup, try using this.

Comment (xerus2000) [reply source/chapters/appendix.rst:317]: Should I leave the list below that in?

**Comment by xerus2000** [comment]
Btw, I have no problem with anyone performing direct edits on this branch, this is not my baby, just something I stumbled upon ^^

**Comment by xerus2000** [comment]
@Be-ing  :)

**Comment by Holzhaus** [comment]
@xerus2000 Thanks!

**Pull request merged**